ban/tests/test_validators.py

```diff
@@ -199,6 +199,33 @@ def test_cannot_create_group_without_kind(session):
     assert validator.errors
 
 
+def test_can_create_group_with_fantoir_on_9digits(session):
+    municipality = MunicipalityFactory(insee="12345")
+    validator = models.Group.validator(name='Rue des Girafes',
+                                       kind=models.Group.WAY,
+                                       municipality=municipality,
+                                       fantoir='123456789')
+    assert not validator.errors
+
+
+def test_can_create_group_with_fantoir_on_10digits(session):
+    municipality = MunicipalityFactory(insee="12345")
+    validator = models.Group.validator(name='Rue des Girafes',
+                                       kind=models.Group.WAY,
+                                       municipality=municipality,
+                                       fantoir='1234567890')
```

Comment:
Erf, this should not pass. We a have a problem, Houston!
Given we have set the `max_length` to 9, Cerberus should have raised an error here. Needs investigation, but seems like there is a bug (not linked to this PR).